### PR TITLE
docs: submit proposals to change parameters using both sdk cli apis

### LINKED ISSUE #1055: Add docs for submitting proposals to changes params
we should add some docs with information on how to submit proposals to such things

fortunatley, there is a lot of information already surrounding this topic in https://docs.cosmos.network/v0.46/modules/gov/01_concepts.html#vote, examples such as https://github.com/celestiaorg/celestia-app/blob/32d247971386c1944d44bec1faeb000b1ff1dd51/app/test/std_sdk_test.go#L260-L276, and our specs https://github.com/celestiaorg/cosmos-sdk/blob/release/v0.46.x-celestia/x/gov/spec/03_messages.md#proposal-submission

cc @adlerjohn and @jacobarluck please add more context if needed :slightly_smiling_face:

### PR DESCRIPTION
**Pull request by evan-forbes**
## Overview

closes #1055

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by evan-forbes** [comment]
these instruction were tested on mocha-4,
Screenshot from 2023-09-13 22-41-32

File: docs/nodes/celestia-app-commands.md [hunk 1]
```diff
@@ -153,6 +153,70 @@ the following command:
 celestia-appd tx gov vote <proposal id> <yes or no> --from <wallet> --chain-id <chain-id>
 ```
 
+Governance proposals on Celestia are limited as there are no text proposals,
+upgrades occur via social consensus, and some only params are not modifiable.
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:157]:
```suggestion
major upgrades occur via social consensus, and some parameters are not modifiable through on-chain governance.
```

**Comment by jcstein** [comment]
thanks for this @evan-forbes

[not blocking] add links from the issue for anyone who wants more context

File: docs/nodes/celestia-app-commands.md [hunk 2]
```diff
@@ -153,6 +153,70 @@ the following command:
...
+However, one can submit a governance proposal to change certain parameters using
+the following instructions.
+
+To change the max validators to 105, one would first save this JSON file:
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:161]:
```suggestion
As an example, to change the max validators to 105, first create this JSON file:
```

File: docs/nodes/celestia-app-commands.md [hunk 3]
```diff
@@ -153,6 +153,70 @@ the following command:
...
+      "subspace": "staking",
+      "key": "MaxValidators",
+      "value": 105
+    }
+  ],
+  "deposit": "1000000000utia"
+}
+```
+
+Then you can submit the proposal with:
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:178]:
```suggestion
Then submit the governance proposal with:
```

File: docs/nodes/celestia-app-commands.md [hunk 4]
```diff
@@ -153,6 +153,70 @@ the following command:
...
+
+```sh
+celestia-appd tx gov submit-legacy-proposal parameter-change <path to json file> --from <wallet> --chain-id <chain-id>
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:181]: What's the chain ID? Can you link to the possible values for different chains? Also link to how to choose/create wallets.

File: docs/nodes/celestia-app-commands.md [hunk 5]
```diff
@@ -153,6 +153,70 @@ the following command:
...
+```
+
+Note that the "legacy" proposal type here is submitting the same type of
+proposal as the normal `submit-proposal` command under the hood. It is the
+reccomended way to submit a proposal via the CLI. If we want to use the newer
+api, we can submit a proposal by first saving the `sdk.Msg` proposal in the json
+encoded format to a json.
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:188]:
```suggestion
API, we can submit a proposal by first saving the `sdk.Msg` proposal in JSON format.
```

File: docs/nodes/celestia-app-commands.md [hunk 6]
```diff
@@ -153,6 +153,70 @@ the following command:
...
+
+```json
+{
+    "messages": [
+        {"@type":"/cosmos.gov.v1beta1.MsgSubmitProposal",
+        "content":{
+            "@type":"/cosmos.params.v1beta1.ParameterChangeProposal",
+            "title":"title",
+            "description":"description",
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:197]:
```suggestion
            "title":"Staking Param Change",
            "description":"Update max validators",
```

File: docs/nodes/celestia-app-commands.md [hunk 7]
```diff
@@ -153,6 +153,70 @@ the following command:
...
+            "changes":[{"subspace":"staking","key":"MaxValidators","value":"103"}]
```
Comment (adlerjohn) [inline docs/nodes/celestia-app-commands.md:198]:
```suggestion
            "changes":[{"subspace":"staking","key":"MaxValidators","value":"105"}]
```

**Review by adlerjohn — COMMENTED** [review]
Can you also add instructions for voting (along with the different options for voting) and querying the status of a proposal?

File: docs/nodes/celestia-app-commands.md (hunk 4 above)
Comment (evan-forbes) [reply docs/nodes/celestia-app-commands.md:187]: sure updated in https://github.com/celestiaorg/docs/pull/1058/commits/4c1ecfabdfee3b708916bf26902f8cb838885405

File: docs/nodes/celestia-app-commands.md [hunk 8]
```diff
@@ -153,6 +153,76 @@ the following command:
 celestia-appd tx gov vote <proposal id> <yes or no> --from <wallet> --chain-id <chain-id>
 ```
 
+>Note: Similar to other cli commands on this page, the chain-id should be for the
```
Comment (jcstein) [inline docs/nodes/celestia-app-commands.md:156]:
```suggestion
>Note: Similar to other CLI commands on this page, the chain-id should be for the
```

File: docs/nodes/celestia-app-commands.md [hunk 9]
```diff
@@ -153,6 +153,76 @@ the following command:
...
+>network that the transaction is targeting. This can be found by querying the
+>same rpc endpoint for the most recent header or simply by looking at a block
```
Comment (jcstein) [inline docs/nodes/celestia-app-commands.md:158]:
```suggestion
>same RPC endpoint for the most recent header or simply by looking at a block
```

**Comment by evan-forbes** [comment]
> Expand the instructions for voting to include the different options for voting, and
Add instructions for querying the status of a proposal?

added in https://github.com/celestiaorg/docs/pull/1058/commits/a429d946e73b64048d204389f08cbcbc81c1cfee

**Review by jcstein — APPROVED** [review]
LGTM

**Pull request merged**